### PR TITLE
Added a vlog for OSD: Make meaningful contribution at OSD

### PR DESCRIPTION
**Pull request by PrarthonaPaul**

### CODE REVIEW

File: _posts/2023-09-13-make-meaningful-contributions-at-osd.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: 'Vlog: Join us at Open Source Day on September 22nd, 2023 and Level Up Your Skills'
+date: 2023-09-13
+tags: ghc ghc23 osd opensourceday
+synopsis: Check out this short video to learn how you can join the WildFly Elytron project and make meaningful contributions to the open source community!
+author: szaldana
```
Comment (jmesnil) [inline _posts/2023-09-13-make-meaningful-contributions-at-osd.adoc:7]: should be `soniazaldana` 

https://github.com/wildfly/wildfly.org/blob/1a8251669d1458fa9e8215e047c6948d7f70016f/_data/authors.yaml#L123C1-L123C13

**Review by jmesnil — CHANGES_REQUESTED** [review]

File: _posts/2023-09-13-make-meaningful-contributions-at-osd.adoc [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: 'Vlog: Join us at Open Source Day on September 22nd, 2023 and Level Up Your Skills'
```
Comment (jmesnil) [inline _posts/2023-09-13-make-meaningful-contributions-at-osd.adoc:3]: title seems incorrect... should it be "Make Meaningful Contributions at Open Source Day" instead ?

Comment (PrarthonaPaul) [reply _posts/2023-09-13-make-meaningful-contributions-at-osd.adoc]: Oh, Sorry about that. Both of these should be fixed now! 
Thanks for pointing it out!

**Review by jmesnil — APPROVED** [review]

**Comment by jmesnil** [comment]
@PrarthonaPaul thanks!

**Pull request merged**